src/assets/country.css

```diff
@@ -0,0 +1,61 @@
+.cuntries-container {
+  display: flex;
+  justify-content: center;
+  padding: 2rem;
+  flex-wrap: wrap;
+}
+
+.country-item {
+  width: 25%;
+  margin: 0.5rem;
+  text-align: center;
+  padding: 1rem;
+  color: #fff;
+  position: relative;
+}
+
+.country-item:nth-child(even) {
+  background-color: rgb(111, 255, 0);
+}
+
+.country-item:nth-child(odd) {
+  background-color: rgb(0, 195, 255);
+}
```

Comment:
- [ ] You did a great job however it is required to follow the given design. Kindly select one main color for the app and be consistent , use it in different shades as in the Design.